vite.config.js

```diff
@@ -55,15 +55,22 @@ export default defineConfig(({ command, mode }) => {
       ...(env.SENTRY_ORG && env.SENTRY_PROJECT && env.SENTRY_AUTH_TOKEN
         ? [
             sentryVitePlugin({
-              include: 'dist/assets/',
-              urlPrefix: '~/assets/',
               org: env.SENTRY_ORG,
               project: env.SENTRY_PROJECT,
               authToken: env.SENTRY_AUTH_TOKEN,
-              release: env.PUBLIC_GIT_COMMIT_HASH,
-              setCommits: {
-                repo: 'greenriver/hmis-frontend',
-                commit: env.FULL_GITHASH,
+              release: {
+                uploadLegacySourcemaps: {
+                  paths: ['dist/assets/'],
+                  urlPrefix: '~/assets/',
+                },
```

Comment:
tried to preserve this config from the previous version of the vite sentry plugin (0.6)